src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -2001,6 +2175,7 @@ struct Simplex_tree_options_full_featured {
   static const bool store_key = true;
   static const bool store_filtration = true;
   static const bool contiguous_vertices = false;
+  static const bool link_nodes_by_label = false;
```

Comment:
This new option needs to be documented in concept/SimplexTreeOptions.h. Maybe we should mention its existence in the doc of star_simplex_range and cofaces_simplex_range.
From its name, it sounds like Simplex_tree_options_full_featured should have this option (otherwise it gets as confusing as intmax_t being shorter than int128_t). On the other hand, a number of current uses of Simplex_tree_options_full_featured don't need the new option. That's something that requires some thought, so if we merge this branch, I want a blocking issue to force us to settle that before the next release.

Reply:
Add some documentation on dd26bff